packages/amplify_api/test/graphql_helpers_test.dart

```diff
@@ -787,9 +787,9 @@ void main() {
 
   group('without ModelProvider', () {
     test('should handle no ModelProvider instance', () {
-      AmplifyAPI api = AmplifyAPI();
+      AmplifyAPI();
```

Comment:
Prefer to delete

Reply:
needed to test failure logic of not using model provider. If deleted, uses previously registered one.